python/stempy/image/__init__.py

```diff
@@ -392,17 +392,21 @@ def maximum_diffraction_pattern(reader, darkreference=None):
     return img
 
 
-def com_sparse(array, crop_to=None):
+def com_sparse2(array, crop_to=None, init_center=None, replace_nans=True):
```

Comment:
I don't think we need to rename? The defaults for the new parameters should give the old behavior?

Reply:
That is a mistake. I was testing it in another file and forgot to change it back. I just pushed a fix.